.storybook/preview.ts

```diff
@@ -1,4 +1,6 @@
 import type { Preview } from "@storybook/react";
+import "primereact/resources/primereact.min.css";
+import "primereact/resources/themes/saga-blue/theme.css";
```

Comment:
use this theme instead.
`import 'primereact/resources/themes/lara-light-indigo/theme.css';`